### DISCUSSION TITLE
What does mean the sample code?

### DISCUSSION THREAD

**Discussion by dimat2** [d3/d3#3822] [Q&A] [answered]
const series = d3.stack()
      .keys(d3.union(this.data.map(d => d.age))) // distinct series keys, in input order
      .value(([,D], key, ) => D.get(key).population) // get value for each series key and stack
      (d3.index(this.data, d => d.state, d => d.age)); // group by stack then series key

I don't understand what does mean the value() function?

Please help.

**Comment by dimat2** [comment]
![Névtelen](https://github.com/d3/d3/assets/40808548/6b9218ba-66bb-4903-a1d6-790dfda5e458)

**Comment by dimat2** [comment]
![Névtelen](https://github.com/d3/d3/assets/40808548/53b04132-f0a4-4e01-9ce0-c2878dcf8c5d)

**Comment by ghost** [comment]
Math.Pi is somehow windows security number You will any time for
negotiation of Project redirected if any loss by SicSec security
Lg

На пон., 05 фев. 2024 г. 19:55, dimat2 ***@***.***> напиша:

**Comment by Fil** [comment, marked as answer]
it is documented here: https://d3js.org/d3-shape/stack#stack_value

**Comment by ghost** [comment]
You are trying to make conflicts, dear

На пон., 05 фев. 2024 г. 21:01, Philippe Rivière ***@***.***>
напиша:
